README.md

```diff
@@ -407,6 +410,51 @@ public interface ModelServerClientApiV2<A> {
 }
 ```
 
+<details>
+	<summary>v2 Subscription Options API</summary>
+```java
```

Comment:
```suggestion
	<summary>v2 Subscription Options API</summary>

```java
```

Reply:
Weird, it works fine in Marked.app! Which md previewer do you use, so I can avoid this next time?

I force-pushed commit 281c63400c95c562f64187878f9f39560402f85b over the previous commit to fix this. Thanks!